### PR TITLE
Make footer year dynamic

### PR DESCRIPTION
**Pull request by steven-tey**
...so you don't have to manually update it every year :)

Let me know if this makes sense, feel free to close if it doesn't!

### CODE REVIEW

File: packages/cli/src/generator_templates/js/emails/components/Footer.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function Footer() {
           fontSize={textSm}
           color={grayDark}
         >
-          © 2022 BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
+          © ${new Date().getFullYear()} BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
```
Comment (psugihara) [inline packages/cli/src/generator_templates/js/emails/components/Footer.jsx:14]:
```suggestion
          © {new Date().getFullYear()} BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
```

File: packages/cli/src/generator_templates/ts/emails/components/Footer.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function Footer() {
           fontSize={textSm}
           color={grayDark}
         >
-          © 2022 BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
+          © ${new Date().getFullYear()} BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
```
Comment (psugihara) [inline packages/cli/src/generator_templates/ts/emails/components/Footer.tsx:14]:
```suggestion
          © {new Date().getFullYear()} BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
```

**Review by psugihara — CHANGES_REQUESTED** [review]
love it! though I think you need to get rid of the `$`'s

**Comment by steven-tey** [comment]
whoops lol you're right! :)

**Comment by psugihara** [comment]
awesome. thanks @steven-tey !

**Pull request merged**